### ISSUE TITLE
Stopped working

### ISSUE THREAD

**Issue by pavelsalamon** [OsaSoft/youtube-better-subscriptions#74]
Hi, the extension stopped working a day ago.

The overlay icon to mark video as watched does not display anymore, and the watched videos are not hidden.

This is output from the console:
```
[YT-Better-Subs] Polymer detected: true util.js:11:17
[YT-Better-Subs] Initializing... util.js:11:17
[YT-Better-Subs] Found page loader util.js:11:17
[YT-Better-Subs] Page was changed to /feed/subscriptions util.js:11:17
[YT-Better-Subs] Stopping subs page behaviour util.js:11:17
[YT-Better-Subs] Initializing subs page... util.js:11:17
[YT-Better-Subs] Building subs UI util.js:11:17
[YT-Better-Subs] Adding element to menu UI 4 util.js:11:17
[YT-Better-Subs] Removing watched from feed and adding overlay util.js:11:17
[YT-Better-Subs] ERROR!  dismissableDiv.querySelector is not a function util.js:16:13
removeWatchedAndAddButton@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs-ui.js:255:32
initSubs@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs.js:140:5
handlePageChange@moz-extension://53c36459-5360-4151-b933-2adcfec55865/pageHandler.js:27:21
initPageHandler@moz-extension://53c36459-5360-4151-b933-2adcfec55865/pageHandler.js:62:29
util.js:17:13
[YT-Better-Subs] Removing watched from feed and adding overlay util.js:11:17
[YT-Better-Subs] ERROR!  dismissableDiv.querySelector is not a function util.js:16:13
removeWatchedAndAddButton@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs-ui.js:255:32
initSubs/intervalId<@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs.js:133:17
util.js:17:13
[YT-Better-Subs] Removing watched from feed and adding overlay util.js:11:17
[YT-Better-Subs] ERROR!  dismissableDiv.querySelector is not a function util.js:16:13
removeWatchedAndAddButton@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs-ui.js:255:32
initSubs/intervalId<@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs.js:133:17
util.js:17:13
[YT-Better-Subs] Removing watched from feed and adding overlay util.js:11:17
[YT-Better-Subs] ERROR!  dismissableDiv.querySelector is not a function util.js:16:13
removeWatchedAndAddButton@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs-ui.js:255:32
initSubs/intervalId<@moz-extension://53c36459-5360-4151-b933-2adcfec55865/subs.js:133:17
```

The same error message keeps repeating.

Any idea what's going on? Did youtube change something that causes this?

**Comment by Phantom1472** [comment]
Hello, 

I am having the same problem.

**Comment by OsaSoft** [comment]
Duplicate of #75, resolved with #76 

A recent Youtube update broke the extension, fix is done and released. Atm Firefox version is already available, Chrome version is pending review, but should be rolling out soon (hopefully)

**Comment by OsaSoft** [comment]
(just noticed this was actually posted before #75, so that should have been the duplicate. Sorry)
